src/main/java/org/openrewrite/jenkins/github/ReplaceSequenceVisitor.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.jenkins.github;
+
+import org.openrewrite.Cursor;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.marker.Markers;
+import org.openrewrite.yaml.JsonPathMatcher;
+import org.openrewrite.yaml.YamlIsoVisitor;
+import org.openrewrite.yaml.tree.Yaml;
+
+import java.util.List;
+
+import static org.openrewrite.Tree.randomId;
+
+/**
+ * Adapted from {@link org.openrewrite.yaml.AppendToSequenceVisitor}
+ */
+class ReplaceSequenceVisitor extends YamlIsoVisitor<ExecutionContext> {
```

Comment:
Glad to see you were able to make this work; I'm wondering if it would make sense to pull this up, or change either `AppendToSequence` or `ChangePropertyValue` to be able to set a new list of values. Just a thought for now, not something I'd ask of you, but curious how you'd see that. Fine to keep this here for now, and know that it's here if we get similar requests.

Reply:
I do think that would make sense. I actually checked both those recipes to see if they already did this.

I think it might make sense for `AppendToSequence` to have a strategy called `REPLACE`, but it definitely seems like `ChangePropertyValue` could take a list of values. I attempted to use `ChangePropertyValue` with a yaml list `[ a, b, c ]`, but the formatting didn't quite work.